docs/encyclopedia/index.mdx

```diff
@@ -29,6 +29,7 @@ The following Encyclopedia pages describe the concepts, components, and features
 - [Temporal Service](/clusters)
 - [Visibility](/visibility)
 - [Namespaces](/namespaces)
+- [Temporal Nexus](/temporalnexus)
```

Comment:
@bergundy I like that it's shorter. If Nexus in the context of the Temporal always means Temporal Nexus (and not Nexus RPC), then I could see doing this, but IMO this means in the context of Temporal we'd need to:
1. consistently introduce and refer to **Nexus as a feature of the Temporal platform** designed to connect durable executions across team, namespace, region, and cloud boundaries.
2. keep Nexus RPC distinct from Temporal's Nexus feature, and framed that way: Nexus RPC SDK, Nexus RPC endpoint ... see [related convo](https://github.com/temporalio/documentation/pull/3046#discussion_r1755986964) re: how a Temporal Nexus Endpoint does a lot more than a basic Nexus RPC endpoint and wanting to distinguish the two.

Outside the context of Temporal, would still use Temporal Nexus when referring to Nexus as a feature of the Temporal platform.

Reply:
Since we had a convo via slack on this, resolving. In terms of the slugs, the evaluate page will be /nexus, encyclopedia will be /temporal-nexus, and cloud will be /cloud/nexus.